editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/features/create/ext/CreateEndpointFigureFeature.java

```diff
@@ -94,4 +94,13 @@ protected String getIconName() {
 		}
 		return super.create(context);
 	}
+
+	/**
+	 *  /!\ Public for test purpose only
+	 * @return
+	 */
+	@Deprecated
+	public List<Dependency> getDeps() {
```

Comment:
I know its marked deprecated but can we still use better naming for that method like getDependencies?